app/services/courses/generate_course_title_service.rb

```diff
@@ -0,0 +1,54 @@
+module Courses
+  class GenerateCourseTitleService
+    def execute(course:)
+      subjects = course.subjects
+
+      title = if course.further_education_course?
+                further_education_title
+              elsif is_modern_lanuage_course?(subjects)
+                modern_language_title(subjects)
+              else
+                generated_title(subjects)
+              end
+
+      title = append_send_info(title) if course.is_send?
```

Comment:
Physics **with** Psychology **with** Special educational needs and disability

?

Reply:
It should be something like "Physics with Pyschology (SEND)"

Reply:
ummm....

Modern Languages (English) (SEND)

?

Reply:
@danielburnley confirmed with @fofr its fine 

>Modern Languages (English) (SEND)
Modern Languages (SEND)

with those as the use case should be relatively low and the user still retain the ability to request their own title

thro work like to have specs to explicitly test it as a way of documentation that was considered